packages/cli/commands/sandbox/sync.js

```diff
@@ -0,0 +1,251 @@
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logger } = require('@hubspot/cli-lib/logger');
+const Spinnies = require('spinnies');
+const { initiateSync } = require('@hubspot/cli-lib/sandboxes');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logErrorInstance } = require('@hubspot/cli-lib/errorHandlers');
+const { ENVIRONMENTS } = require('@hubspot/cli-lib/lib/constants');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { getAccountConfig, getConfig, getEnv } = require('@hubspot/cli-lib');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cli-lib/lib/urls');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { uiLine } = require('../../lib/ui');
+const {
+  getAccountName,
+  getSyncTypes,
+  pollSyncStatus,
+} = require('../../lib/sandboxes');
+const {
+  isMissingScopeError,
+} = require('@hubspot/cli-lib/errorHandlers/apiErrors');
+
+const i18nKey = 'cli.commands.sandbox.subcommands.sync';
+
+exports.command = 'sync';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  await loadAndValidateOptions(options);
+
+  const { force } = options;
+  const config = getConfig();
+  const accountId = getAccountId(options);
+  const accountConfig = getAccountConfig(accountId);
+  const spinnies = new Spinnies({
+    succeedColor: 'white',
+  });
+
+  trackCommandUsage('sandbox-sync', null, accountId);
+
+  if (
```

Comment:
I think the code here is very understandable, and I could read through it with relative ease. For convenience's sake, though, could you add a few comments to clarify some of these conditionals?

Reply:
For sure!